.github/workflows/vulnerability-scan.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         echo "::set-env name=IMAGE_WITH_POETRY_LOCK::$IMAGE_NAME"
 
     - name: Scan image 🕵️‍♀️🕵️‍♂️
-      uses: homoluctus/gitrivy@v2.0.0
+      uses: wochinge/gitrivy@master
```

Comment:
might make sense to tag this commit so that we'll know it's based on `2.0.0`

Reply:
I'll reference the SHA, ok? Hopefully this gets merged in the official action one soon.

Reply:
👍